cinema/serializers.py

```diff
@@ -69,6 +96,7 @@
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = serializers.ImageField(source="movie.image", read_only=True)
```

Comment:
The `MovieSessionListSerializer` correctly includes the `movie_image` field, ensuring that the image URL is displayed in the movie session list.